rules/no-ignored-test-files.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+var path = require('path');
+var pkgUp = require('pkg-up');
+var multimatch = require('multimatch');
+var util = require('../util');
+var createAvaRule = require('../create-ava-rule');
+
+var defaultFiles = [
+	'test.js',
+	'test-*.js',
+	'test/**/*.js'
+];
+
+function isInIgnoredFolder(relativeFilePath) {
+	var folders = relativeFilePath.split(path.sep);
+	var ignoredFolder;
+	if (folders.indexOf('fixtures') !== -1) {
+		ignoredFolder = 'fixtures';
+	} else if (folders.indexOf('helpers') !== -1) {
+		ignoredFolder = 'helpers';
+	}
+
+	if (ignoredFolder) {
+		return 'Test file is ignored because it is inside a `' + ignoredFolder + '` folder';
+	}
+
+	return null;
+}
+
+function isNotInGlob(files, relativeFilePath) {
+	if (multimatch([relativeFilePath], files).length === 0) {
+		return 'Test file is ignored because it is not in `' + files.join(' ') + '`';
+	}
+
+	return null;
+}
+
+function isIgnored(rootDir, files, filepath) {
+	var relativeFilePath = path.relative(rootDir, filepath);
+	return isInIgnoredFolder(relativeFilePath) ||
+		isNotInGlob(files, relativeFilePath);
+}
+
+function getPackageInfo() {
+	var packageFilePath = pkgUp.sync();
```

Comment:
I'll change the wording to indicate that it will use the closest package.json. I think in most cases having multiple `package.json` will be used for examples inside the repo or test fixtures (though not sure I see the value in the one you showed).

> I wonder if you should keep searching until you find AVA config?

Why not, but then again, it will not be able to handle some cases. Some projects might have sub-projects as examples (thinking of [Redux](https://github.com/reactjs/redux/tree/master/examples, even though it doesn't use AVA afaik) where it's pretty much the point that they don't depend on any parent dirs. I'm guessing that in most cases, what you want is isolation from the parent "project", so I'd rather not do this. But let me know what you guys think.